src/pages/SwapV2/index.tsx

```diff
@@ -629,17 +629,19 @@ export default function Swap({ history }: RouteComponentProps) {
     if (isSelectCurencyMannual) syncUrl(currencyIn, currencyOut) // when we select token manual
   }, [currencyIn, currencyOut, isSelectCurencyMannual, syncUrl])
 
+  const chainIdDebounce = usePrevious(chainId) || chainId
```

Comment:
name `Debounce` make me confuse. Name it `chainIdPrevious`

Reply:
ah, sorry. I forgot rename it. updated